src/content/docs/guides/first-command.mdx

```diff
@@ -48,7 +48,7 @@ const client = new Client();
 client.start().then(() => client.uploadCommands());
 ```
 
-Pretty simple, right? But sometimes commands are not just about receiving a request and responding to it, you also have to think about what the user wants, that's where the `options` come in, let's make the response of this command something private.
+Pretty simple, isn't it? But sometimes commands are not just about receiving a request and responding to it, you also have to think about what the user wants, that's where the `options` come in, let's make the response of this command something private.
```

Comment:
"right" in this context is correctly. Because if we use "isn't it" means we are not sure that it is simple

Reply:
isn't it refers to right. The same thing but more polite. https://hinative.com/questions/15039892